content/hardware/04.pro/boards/portenta-h7/tutorials/getting-started-openmv-micropython/content.md

```diff
@@ -162,14 +162,14 @@ In this tutorial you learned how to use the OpenMV IDE with your Portenta board.
 ### Next Steps
 
 -   Experiment with MicroPythons capabilities. If you want some examples of what to do, take a look at the examples included in the OpenMV IDE. Go to: **File > Examples > Arduino > Portenta H7** in the OpenMV IDE.
--   Take a look at our other Portenta H7 tutorials which showcase its many uses. You can find them [here](https://docs.arduino.cc/hardware/portenta-h7#tutorials).
+-   Take a look at your other Portenta H7 tutorials which showcase its many uses. You can find them [here](https://docs.arduino.cc/hardware/portenta-h7#tutorials).

```

Comment:
I see `our` better than `your`, but we can switch it with `the` better?

Reply:
Definetely!

Reply:
closing this, change addressed